src/init.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python
+
+import argparse
+import os
+
+from kubernetes import client, config
+from kubernetes.client import ApiClient, Configuration
+
+from openshift.dynamic import DynamicClient
+
+parser = argparse.ArgumentParser(description="Options to Program")
+parser.add_argument('-n', default="openshift-machine-api", dest="namespace", help="Namespace to use")
+parser.add_argument('-c', default="/config/env/CLUSTERID", required=False, dest="clusteridfile", help="Location to write CLUSTERID")
+parser.add_argument('-r', default="/secrets/aws/config.ini", required=False, dest="regionfile", help="Location to write AWS region config.ini")
+parser.add_argument('-o', default="/secrets/aws/credentials.ini", required=False, dest="credsfile", help="Location to write AWS credentials.ini")
+parser.add_argument('-a', default="/secrets/aws_access_key_id", required=False, dest="accesskeyinfile", help="Source of AWS access key")
+parser.add_argument('-A', default="/secrets/aws_secret_access_key", required=False, dest="secretkeyinfile", help="Source of AWS secret key")
+
+args = vars(parser.parse_args())
+
+def ensure_dir(filepath):
+    if not os.path.exists(os.path.dirname(filepath)):
+        os.makedirs(os.path.dirname(filepath))
+
+def get_cluster_id(client, namespace):
+    v1b_machines = client.resources.get(api_version='machine.openshift.io/v1beta1', kind='Machine')
```

Comment:
Will have to change `api_version` at some point, this will go to v1.  If version is required I wonder if the bash script would have been simpler.  Wouldn't have needed to know version, just grab the json and parse it.

Reply:
@jewzaam Good pointer; we don't need to. See 6eddf2b.